app/serializers/event_date_serializer.rb

```diff
@@ -30,4 +30,11 @@ class EventDateSerializer < ActiveModel::Serializer
   attribute :date do
     Date.parse(object.date.to_s)
   end
+
+  attribute :event do
```

Comment:
This works but if I understand correctly it seems our JSON will be repeating itself:

event
&nbsp;&nbsp;event_date
&nbsp;&nbsp;&nbsp;&nbsp;event

The event_date JSON contains an event_id data element.  It am thinking it would be better to add a show element to the event controller to return the elements from the event that you need to accomplish your task. It is an extra request but here we have two serializers doing the same thing.  And if you need additional data elements from the event you will need them and if there are changes they would be made twice.

If this is for registration I wouldn't guess the extra request would be a performance issue.  If we go this route I would be happy to write the event show method and test it.